.github/EXTERNAL_CONTRIBUTOR_PR_COMMENT.md

```diff
@@ -0,0 +1,5 @@
+Thank you for your contribution to the Noir language.
+
+Please **DO NOT FORCE PUSH TO THIS BRANCH** after the Noir team have reviewed this PR. Doing so will only delay us merging your PR as we will need to start the review process from scratch.
```

Comment:
```suggestion
Please **do not force push to this branch** after the Noir team have reviewed this PR. Doing so will only delay us merging your PR as we will need to start the review process from scratch.
```
I think the bold should be enough. The caps make it sound a bit too aggressive I think